src/components/magasin/panel/index.js

```diff
@@ -193,7 +194,26 @@ export default class StorageUnitContainer extends Component {
     );
   }
 
+  getNodeTypeItems() {
+    // First level 'Organisation'. Second level 'Building'
+    const nodeType = ['StorageUnit', 'Room', 'Building', 'Organisation'];
+    const nodeTypeOrganisation = ['Organisation'];
+    const nodeTypeBuilding = ['Building'];
+    let lv_return;
+    if (this.props.rootNode &&
```

Comment:
const { rootNode } = this.props;
const { pathNames } = rootNode || {}
if (pathNames && pathNames.length > .. ..... ) {
   ....
}